.drawio

```diff
@@ -0,0 +1,281 @@
+<mxfile host="65bd71144e" modified="2020-12-07T04:01:47.630Z" agent="5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Code/1.51.1 Chrome/83.0.4103.122 Electron/9.3.3 Safari/537.36" etag="hazZwUvSiQrNkgPYDqzd" version="13.10.0" type="embed">
```

Comment:
What is this file used for?

Reply:
This file is an editable drawing that can be viewed in VSCode using the Draw.io extension. I thought it would be useful for other teammates if needed. I can remove it if wanted, but I don't know how to from here on github.

Reply:
@Fluellen-Arman  If you want to keep this in the repo, I'd suggest putting it in to a separate PR that puts it inside the /docs subdirectory, along with some explanation in the README.md that points to it.

Otherwise, you should delete it from this commit.    Just checkout the branch, use `git rm .drawio` and then commit that change.